test/unit/prof_threshold.c

```diff
@@ -0,0 +1,103 @@
+#include "test/jemalloc_test.h"
+
+// Test config (set in reset_test_config)
```

Comment:
Here and below in couple of places `//` -> `/**/`, for consistency with other files.

```suggestion
/* Test config (set in reset_test_config) */
```